pyrsa/util/descriptor_utils.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Descriptor handling
+
+@author: heiko
+"""
+
+
+def format_descriptor(descriptors):
+    """ formats a descriptor dictionary
+        Args:
+            descriptors(dict): the descriptor dictionary
+
+        Returns:
+            string_descriptors(String): formated string to show dict
+    """
+    string_descriptors = ''
+    for entry in descriptors:
+        string_descriptors = (string_descriptors +
+                              f'{entry} = {descriptors[entry]}\n'
+                              )
+    return string_descriptors
```

Comment:
how about `return ''.join([f'{k} = {v}\n' for k, v in descriptors.items()])`? Too terse perhaps?

Reply:
well, is it faster or in some other way better than the loop? If so, I think it is hidden enough that we could change to your one-line solution; it is hidden enough after all.

Reply:
No both are fine